docs/docusaurus/docs/guides/validation/checkpoints/how_to_validate_multiple_batches_within_single_checkpoint.md

```diff
@@ -0,0 +1,71 @@
+
+The following Python code will create a Batch Request that includes every available Batch in a Data Asset named `asset`:
+
+```python name="tests/integration/docusaurus/validation/checkpoints/how_to_validate_multiple_batches_within_single_checkpoint.py build_a_batch_request_with_multiple_batches"
+```
+
+:::tip
+A Batch Request can only retrieve multiple Batches from a Data Asset that has been configured to include more than the default single Batch.
+
+When working with a Filesystem Data Source and organizing Batches, the `batching_regex` argument determines the inclusion of multiple Batches into a single Data Asset, with each file that matches the `batching_regex` resulting in a single Batch.
+
+On the other hand, when utilizing a SQL Datasource, Data Assets include a single Batch by default, which can then be split into multiple Batches using splitters.
+
+For more information on partitioning a Data Asset into Batches, see [Manage Data Assets](/docs/guides/connecting_to_your_data/manage_data_assets_lp).
+:::
+
+## Get a list of Batches from the Batch Request
+
+Use the same Data Asset that your Batch Request was built from to retrieve a list of Batches with the following code:
+
+```python name="tests/integration/docusaurus/validation/checkpoints/how_to_validate_multiple_batches_within_single_checkpoint.py batch_list"
+```
+
+## Convert the list of Batches into a list of Batch Requests
+
+Since a Checkpoint validates Batch Requests, but only validates the last Batch found in a Batch Request, the next step in this process is to convert the list of Batches into a list of Batch Requests that return the corresponding individual Batch.
+
+```python name="tests/integration/docusaurus/validation/checkpoints/how_to_validate_multiple_batches_within_single_checkpoint.py batch_request_list"
+```
+
+## Build a validations list 
+
+A Checkpoint class's `validations` parameter consists of a list of dictionaries.  Each dictionary pairs one Batch Request with the Expectation Suite it should be validated against.  The following code creates a valid `validations` list where each Batch Request is associated with an Expectation Suite named `example_suite`.
+
+```python name="tests/integration/docusaurus/validation/checkpoints/how_to_validate_multiple_batches_within_single_checkpoint.py add_validations"
+```
+
+## Run Checkpoint
+
+The `validations` list, containing the pairings of Batch Requests and Expectation Suites, can now be passed to a single Checkpoint instance which will validate each Batch Request against its corresponding Expectation Suite.  This effectively validates each Batch included in the original multiple-Batch Batch Request.
+
+```python name="tests/integration/docusaurus/validation/checkpoints/how_to_validate_multiple_batches_within_single_checkpoint.py add_checkpoint"
+```
+
+## Review the Validation Results
+
+Once the validations have been run, the following code can be used to build and view the Validation Results as Data Docs:
```

Comment:
```suggestion
After the validations run, use the following code to build and view the Validation Results as Data Docs.
```
**Note:** In some instances, you're introducing code samples with colons, and in others you're not. Pick a style and use it consistently.